setup.cfg

```diff
@@ -16,6 +16,7 @@ filterwarnings = error
     ignore:Mixed memory format inputs detected:UserWarning
     ignore:Setting attributes on ParameterDict:UserWarning
     ignore::DeprecationWarning
+    ignore:CUDA initialization:UserWarning
```

Comment:
Works around the spurious warning https://github.com/pytorch/pytorch/issues/47038